mmv1/third_party/terraform/services/iam3/resource_iam_projects_policy_binding_test.go.tmpl

```diff
@@ -0,0 +1,113 @@
+package iam3_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccIAM3ProjectsPolicyBinding_iamProjectsPolicyBindingExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy: testAccCheckIAM3ProjectsPolicyBindingDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIAM3ProjectsPolicyBinding_iamProjectsPolicyBindingExample_full(context),
+			},
+			{
+				ResourceName:            "google_iam_projects_policy_binding.my-project-binding",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"annotations", "location", "policy_binding_id"},
+			},
+			{
+				Config: testAccIAM3ProjectsPolicyBinding_iamProjectsPolicyBindingExample_update(context),
```

Comment:
Could you add a third step where we go back to `testAccIAM3ProjectsPolicyBinding_iamProjectsPolicyBindingExample_full` afterwards?

Reply:
will do